tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Logging/Serilog/Formatting/SerilogLogFormatter.cs

```diff
@@ -0,0 +1,216 @@
+﻿// <copyright file="SerilogLogFormatter.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections;
+using System.Text;
+using Datadog.Trace.DuckTyping;
+using Datadog.Trace.Logging.DirectSubmission;
+using Datadog.Trace.Logging.DirectSubmission.Formatting;
+using Datadog.Trace.Vendors.Newtonsoft.Json;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Logging.Serilog.Formatting
+{
+    internal class SerilogLogFormatter
+    {
+        public static void FormatLogEvent(LogFormatter logFormatter, StringBuilder sb, ILogEvent logEvent)
+        {
+            var message = logEvent.RenderMessage();
+
+            logFormatter.FormatLog(
+                sb,
+                logEvent,
+                logEvent.Timestamp.UtcDateTime,
+                message,
+                eventId: null,
+                GetLogLevelString(logEvent.Level),
+                logEvent.Exception,
+                RenderProperties);
+        }
+
+#if NET45
+        private static string GetLogLevelString(object logLevel) =>
+            (int)logLevel switch
+            {
+                0 => DirectSubmissionLogLevelExtensions.Verbose,
+                1 => DirectSubmissionLogLevelExtensions.Debug,
+                2 => DirectSubmissionLogLevelExtensions.Information,
+                3 => DirectSubmissionLogLevelExtensions.Warning,
+                4 => DirectSubmissionLogLevelExtensions.Error,
+                5 => DirectSubmissionLogLevelExtensions.Fatal,
+                _ => throw new ArgumentOutOfRangeException(nameof(logLevel))
+            };
+#else
+        private static string GetLogLevelString(LogEventLevelDuck logLevel) =>
+            logLevel switch
+            {
+                LogEventLevelDuck.Verbose => DirectSubmissionLogLevelExtensions.Verbose,
+                LogEventLevelDuck.Debug => DirectSubmissionLogLevelExtensions.Debug,
+                LogEventLevelDuck.Information => DirectSubmissionLogLevelExtensions.Information,
+                LogEventLevelDuck.Warning => DirectSubmissionLogLevelExtensions.Warning,
+                LogEventLevelDuck.Error => DirectSubmissionLogLevelExtensions.Error,
+                LogEventLevelDuck.Fatal => DirectSubmissionLogLevelExtensions.Fatal,
+                _ => throw new ArgumentOutOfRangeException(nameof(logLevel))
```

Comment:
Noting no exceptions so we don't forget*

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/1945/commits/7f655a451f63c85b0c2f961e65dd04892e996920